specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-11-01-preview/Recommendations.json

```diff
@@ -0,0 +1,564 @@
+          }
+        },
+        "tags": [
+          "recommendations"
+        ],
+        "description": "Gets a recommendation by its id.",
+        "operationId": "Get_SingleRecommendation",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/RecommendationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Recommendation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "x-ms-examples": {
+          "Creates a recommendation.": {
+            "$ref": "./examples/recommendations/PatchRecommendation.json"
+          }
+        },
+        "tags": [
+          "recommendations"
+        ],
+        "description": "Patch a recommendation.",
+        "operationId": "Update_Recommendation",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/RecommendationId"
+          },
+          {
+            "$ref": "#/parameters/RecommendationPatch"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "202": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Recommendation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Recommendation": {
```

Comment:
>Recommendation [](http://example.com/codeflow?start=5&length=14)

This needs to be defined as a ProxyResource in ARM

Reply:
Could you provide an example/docs? There are no ProxyReosurces in securityinsights